### ISSUE TITLE
[UI and Data Hookup] Add last updated date for each section of the application

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/dac-ui#495]
## Goal
To add the last update date to each section of the application UI (NOT THE PDF), mostly helpful for admin to see what sections have been revised for a renewal.

API ticket: https://github.com/icgc-argo/dac-api/issues/246

## UI updates
- [ ] Add the last updated text in the title area of each application section
Zeplin: https://zpl.io/Vq48L4Y

![image](https://user-images.githubusercontent.com/18424138/137196541-457204da-f683-4cab-8a8c-ba6c0c7fdb67.png)

**Comment by kcullion** [comment]
Tested in QA, this works as intended. Nice job @anncatton !

- Updated date isn't showing until I make a change to that section 
- Updated date does not show on the pdf
- Updated date changes if I edit an application 
In the example I submitted an application yesterday then approved it today and added a collaborator. The dates update accordingly.

![image.png](https://images.zenhubusercontent.com/5a5f5bcd4b5806bc2bc31a86/8ce0f530-3b41-4317-bb80-05b583991f16)
